app.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const app = express();
-const { getTopics, getEndpoints, getArticlesById } = require("./controllers/ncnewscontrollers");
+const { getTopics, getEndpoints, getArticlesById, getAllArticles } = require("./controllers/ncnewscontrollers");
 
 app.use(express.json());
```

Comment:
We don't need this line of code at the moment, so it can be removed 👍

Reply:
Hi John, thank you! Which line number should I remove?